packages/ckeditor5-table/tests/ui/colorinputview.js

```diff
@@ -205,7 +205,11 @@ describe( 'ColorInputView', () => {
 
 				colorTableView.fire( 'cancel' );
 
-				expect( view.value ).to.be.equal( '#ffaaff' );
+				/**
+				 * TODO, should be HEX format: (https://github.com/ckeditor/ckeditor5/issues/14389)
```

Comment:
You should reference the issue which will handle this TODO, not the one introducing it. Whoever sees this will not get any explanation why this was done.

Reply:
This applies to all the other TODOs as well.